### PR TITLE
bs.js

### PR DESCRIPTION
**Pull request by sabahuddin**
Translation in Bosnian

### CODE REVIEW

**Comment by acerix** [comment]
Great, thank you

File: locale/bs.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import 'moment/locale/ba';
```
Comment (arshaw) [inline locale/bs.js]: `'moment/locale/ba'` doesn't exist ... it would need to be `'moment/locale/bs'`
(causing the build to fail)

could you change this?

Comment (sabahuddin) [reply locale/bs.js]: Done.

**Comment by arshaw** [comment]
released in 3.8.1, thanks

**Pull request merged**